src/components/customers/Customers.jsx

```diff
@@ -1,5 +1,6 @@
 import { Avatar, Container, Typography, Box, Paper } from "@mui/material";
 import Carousel from "react-material-ui-carousel";
+import PropTypes from 'prop-types';
```

Comment:
import order is not in correct order. 
1. react imports

2. library imports

3. local imports